[2025-03-24T12:59:21.181+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:59:21.212+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_2.first_task manual__2025-03-24T12:59:17.673852+00:00 [queued]>
[2025-03-24T12:59:21.227+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_2.first_task manual__2025-03-24T12:59:17.673852+00:00 [queued]>
[2025-03-24T12:59:21.228+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:59:21.251+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-24 12:59:17.673852+00:00
[2025-03-24T12:59:21.262+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=115) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:59:21.277+0000] {standard_task_runner.py:72} INFO - Started process 116 to run task
[2025-03-24T12:59:21.277+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_2', 'first_task', 'manual__2025-03-24T12:59:17.673852+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/fristdags.py', '--cfg-path', '/tmp/tmppdqo5quc']
[2025-03-24T12:59:21.280+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask first_task
[2025-03-24T12:59:21.386+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_2.first_task manual__2025-03-24T12:59:17.673852+00:00 [running]> on host e37514d11bf6
[2025-03-24T12:59:21.639+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-24T12:59:17.673852+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-24T12:59:17.673852+00:00'
[2025-03-24T12:59:21.640+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:59:21.718+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:59:21.721+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'hello this first task'"]
[2025-03-24T12:59:21.751+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:59:21.762+0000] {subprocess.py:106} INFO - hello this first task
[2025-03-24T12:59:21.763+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:59:21.811+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:59:21.811+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_2, task_id=first_task, run_id=manual__2025-03-24T12:59:17.673852+00:00, execution_date=20250324T125917, start_date=20250324T125921, end_date=20250324T125921
[2025-03-24T12:59:21.874+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:59:21.897+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:59:21.899+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
